hypervisor/dm/vioapic.c

```diff
@@ -304,8 +302,8 @@ vioapic_indirect_write(struct acrn_vioapic *vioapic, uint32_t addr,
 	}
 
 	/* redirection table entries */
-	if ((regnum >= IOAPIC_REDTBL) &&
-	    (regnum < (IOAPIC_REDTBL + (pincount * 2U)))) {
+	if ((regnum >= IOAPIC_REDTBL) && (regnum < (IOAPIC_REDTBL + (pincount * 2U)))) {
+		bool status = true;
```

Comment:
Don't call it status. Status = true/false didn't tell us what it is for.

Reply:
rename it to wire_mode_valid